templates/analytics/remote_server_support.html

```diff
@@ -118,6 +118,19 @@ <h4>📅 Current plan information:</h4>
                 </select>
                 <button type="submit" class="btn btn-default support-submit-button">Update</button>
             </form>
+
+            <form method="POST" class="remote-server-form">
+                <b>Modify current plan</b><br />
+                {{ csrf_input }}
+                <input type="hidden" name="remote_server_id" value="{{ remote_server.id }}" />
+                <select name="modify_plan" class="modify-plan-method-select" required>
+                    <option disabled value="" selected>-- select --</option>
+                    <option value="downgrade_at_billing_cycle_end">Downgrade at the end of current billing cycle</option>
+                    <option value="downgrade_now_without_additional_licenses">Downgrade now without creating additional invoices</option>
+                    <option value="downgrade_now_void_open_invoices">Downgrade now and void open invoices</option>
+                </select>
+                <button type="submit" class="btn btn-default support-submit-button">Modify</button>
+            </form>
```

Comment:
Since we'll need very parallel template for these for the `RemoteRealm` type customers, it might be worth thinking about refactoring to make some of these blocks into smaller Jinja2 templates that we can include in multiple places, just passing a few parameters like the class name and value for the hidden input.

(Similarly, most of the big billing details block of the HTML above is just a function  of the `current_plan`, and might be able to be shared)